NEWS.md

```diff
@@ -1,3 +1,8 @@
-# admiralpeds 0.1.0.9001
+# admiralpeds  (development version)
```

Comment:
i think we can skip NEWS updates until we make the first release, take a look at the bottom of NEWS for admiralonco: https://github.com/pharmaverse/admiralonco/blob/main/NEWS.md